res/country_metadata/heightUnit.yml

```diff
@@ -0,0 +1,13 @@
+# Sources:
+# https://en.wikipedia.org/wiki/Metrication#Overview
+# https://en.wikipedia.org/wiki/Imperial_units#Current_use
+default: [m]
+CA: [ft, m]
```

Comment:
Canada should be meter only, see https://en.wikipedia.org/wiki/Comparison_of_MUTCD-influenced_traffic_signs
Or, at least, m first, then ft.